src/services/items/service-api.ts

```diff
@@ -204,6 +205,13 @@ const plugin: FastifyPluginAsync = async (fastify) => {
 
         fastify.register(graaspItemTags);
 
+        await fastify.register(graaspItemPublishPlugin, {
+          publishedTagId: PUBLISHED_TAG_ID,
+          publicTagId: PUBLIC_TAG_ID,
+          graaspActor: GRAASP_ACTOR,
+          explorerHostApi: CLIENT_HOSTS[2].hostname,
```

Comment:
Let's not mention `explorer` as we always change the name lol Let's just go for hostname (it's not the api).
For the value, let's not depend on the index, you can use `.find`